src/controllers/tripsController.js

```diff
@@ -102,19 +103,48 @@ class tripsController {
           fromCountry, fromCity, toCountry, toCity, stops
         },
         user: {
-          userID, gender, passportNumber, managerEmail
+          userID, gender, passportNumber, managerEmail, preferedLang
         }
       } = req;
       const from = `${fromCountry} - ${fromCity}`;
       const to = `${toCountry} - ${toCity}`;
 
       if (stops.length <= 1) {
-        return errorResponse(res, 403, req.i18n.__('multiCityFailure'));
+        return errorResponse(res, 403, setLanguage(preferedLang).__('multiCityFailure'));
       }
 
       await tripHelper.createTrip(userID, passportNumber, gender, from, to, managerEmail, req.body);
 
-      successResponse(res, 201, req.i18n.__('multiCitySuccess'));
+      successResponse(res, 201, setLanguage(preferedLang).__('multiCitySuccess'));
+    } catch (error) {
+      errorResponse(res, 500, error.message);
+    }
+  }
+
+  /**
+   *
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+  static async availRequests(req, res) {
+    try {
+      const {
+        query: {
+          page, limit,
+        },
+        user: {
+          role, email, preferedLang
+        }
+      } = req;
+
+      if (role !== 'manager') {
```

Comment:
@AbdoulNuru I think u should only restrict a requester for I can be a super admin and I manage requests from a manager 
so I should also be able to view requests
\
```suggestion
      if (role === 'requester') {
      // return error message
      }
      else 
      .......
```

Reply:
@blaise82 Thanks but this feature is for availing pending requests to a manager from his direct report, so even if you allow a super admin to continue there won't be any requests to show because he is not managing anyone.

Reply:
ohhh i get it now